examples/shebang.md

```diff
@@ -53,5 +53,5 @@ Start the script by calling it like any other command:
 
 <!----------------------------------------------------------------------------->
 
```

Comment:
We should rename this _hashbang_ as it is preferred over the term _shebang_ these days. as some consider the term gendered.